lib/requirejs/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "_args": [
+    [
+      {
+        "raw": "requirejs",
+        "scope": null,
+        "escapedName": "requirejs",
+        "name": "requirejs",
+        "rawSpec": "",
+        "spec": "latest",
+        "type": "tag"
+      },
+      "/home/bhuvy/UIUCClasses/241CourseStaff/illinois-cs241.github.io/lib"
```

Comment:
Because it is part of the package.json no, but for cleanliness sake I will take it out.